src/main/java/uk/gov/ons/census/casesvc/service/RmCaseUpdatedService.java

```diff
@@ -41,6 +41,8 @@ public void processMessage(ResponseManagementEvent rme, OffsetDateTime messageTi
 
     validateRmCaseUpdated(rmCaseUpdated);
 
+    boolean oaPresent = updatedCase.getOa() != null;
```

Comment:
I probably would have preferred to use `StringUtils.isEmpty()` just in case we have a blank OA for some reason; somehow

Reply:
The null check still needs to be there but I've added isEmpty() to the clause to cover blank strings that aren't null.